### ISSUE TITLE
Attachments not displaying when first "attaching" them

### ISSUE THREAD

**Issue by bagwaa** [hanhdt/vue-trix#142]
Thanks for this plugin, I enjoy using it and has saved me a lot of time, however I am seeing some very strange behaviour even on the most basic of installs.

I will describe my problem below and I have also included a repo which is just a vue-cli installed app (the default one) with Vue-Trix added into the main component.

See the full repository here : https://github.com/bagwaa/vue-trix-bug

# to replicate the bug

## Project setup

```
yarn install
yarn run serve
```

## Bug

My HelloWorld component looks like this, as basic as I could get it for this example.

```
<template>
  <div class="hello">
    <VueTrix v-model="editorContent" />
  </div>
</template>

<script>
import VueTrix from "vue-trix";

export default {
  components: {
    VueTrix
  },
  data() {
    return {
      editorContent: ''
    }
  }
}
</script>
```

I have tried this in Chrome and Firefox on Linux

1. If I click on attach file BEFORE entering any content the image does not display.
2. If I have "some content" which has NO <BR> tags (you can see the actual content in vue chrome dev tools) then attaching images is fine.
3. If I try to attach an image straight after a <BR> tag it fails.

I am really confused by this behaviour, bug? or "holding it wrong?"

**Comment by hanhdt** [comment]
Hi @bagwaa ,

Thank you so much for submitting the issue.
It seem like you are right. I will reproduce and check it soon.

**Comment by hanhdt** [comment]
@bagwaa,
If you use 'drag-n-drop' the file with your scenarios, this behaviour won't occurs.
It's only on clicking the attach button, right?

**Comment by bagwaa** [comment]
>@bagwaa, ...

@hanhdt that’s correct, the drag functionality seems to work fine, it’s just when you click on the icon to attach an image.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by abrhambas01** [comment]
Is this related to the issue that I also have when I render the values in the database to the trix editor like this 

![image](https://user-images.githubusercontent.com/12043306/75682948-34ae9b80-5cd1-11ea-8ef0-44f2a2cefa7c.png)

I get that .. Is it something that you folks have encountered. ?

**Comment by inspire22** [comment]
Any progress with this?

**Comment by JohnyJohny1861** [comment]
How could I disable file uploader please help with config object

**Comment by hanhdt** [comment]
@JohnyJohny1861 Can you take a look at this solution?
https://github.com/hanhdt/vue-trix/issues/87

**Comment by Alex-Casale** [comment]
@hanhdt 
I found the same problem.
**Premise:** the serve side loading works.
Pressing the button does not display the image while drag and drop works perfectly

**Comment by hanhdt** [comment]
@Alex-Casale,
Thanks for your clarification.

**Comment by Alex-Casale** [comment]
@hanhdt 
Is there a solution to this bug?
